cli/flox-rust-sdk/src/models/container_builder.rs

```diff
@@ -78,7 +78,31 @@ mod tests {
         let container_builder = ContainerBuilder::new(test_script);
 
         let mut buf = Vec::new();
-        container_builder.stream_container(&mut buf).unwrap();
+        // OS error 26 is "Text file busy",
+        // which can happen when executing a script
+        // that is has been written to immediately before.
+        // We typically see this in tests, where we write
+        // a new script and immediately execute it.
+        // In production use, this should not happen as the script
+        // will be written by a different process (`pkgdb`).
+        //
+        // https://github.com/rust-lang/rust/issues/114554
+        let mut tries = 0;
+        loop {
+            if tries >= 3 {
+                panic!("Calling the container builder script failed too many times: with 'Text file busy'")
+            }
+            match container_builder.stream_container(&mut buf) {
+                Err(ContainerBuilderError::CallContainerBuilder(e))
+                    if e.raw_os_error() == Some(26) =>
```

Comment:
Use an enum if you can instead of `26`

Reply:
https://doc.rust-lang.org/std/io/enum.ErrorKind.html#variant.ExecutableFileBusy

https://github.com/rust-lang/rust/issues/86442

Reply:
pulled it out into a separate constant at least